src/libraries/System.Reflection/tests/AssemblyTests.cs

```diff
@@ -160,6 +160,35 @@ public void GetFile()
             Assert.Equal(typeof(AssemblyTests).Assembly.GetFile("System.Reflection.Tests.dll").Name, typeof(AssemblyTests).Assembly.Location);
         }
 
+        [Fact]
+        public void GetFile_InMemory()
+        {
+            var inMemBlob = File.ReadAllBytes(SourceTestAssemblyPath);
+            var asm = Assembly.Load(inMemBlob);
+            var methods = asm.GetType("GetFileMethods");
+            var getFile = methods.GetMethod("GetFile");
```

Comment:
I don't understand why we have to call the `GetFile` from within the assembly in question. Much simpler repro/test is to just call `asm.GetFile(...)` directly. There's no difference in doing so either way.

Reply:
Ah yeah, that's a good point. Not sure what I was thinking.